### PR TITLE
Added option to choose primusrun or optirun/virtualgl

### PR DESCRIPTION
**Pull request by juliotux**
Give the user the power to choose if use primusrun or optirun/virtugalgl to run games in Optimus laptops.

Primusrun is having some problems for a lot of applications. In other hand, shows better performance than virtualgl.

### CODE REVIEW

File: lutris/game.py [hunk 1]
```diff
@@ -261,9 +261,16 @@ def do_play(self, prelaunched, _error=None):
         # Command
         launch_arguments = gameplay_info['command']
 
-        primusrun = system_config.get('primusrun')
-        if primusrun and system.find_executable('primusrun'):
+        # primusrun = system_config.get('primusrun')
```
Comment (strycore) [inline lutris/game.py]: There is commented code.

File: lutris/sysoptions.py [hunk 2]
```diff
@@ -73,16 +82,28 @@ def get_dri_prime():
         'help': ("Some games don't correctly restores gamma on exit, making "
                  "your display too bright. Select this option to correct it.")
     },
+    # {
```
Comment (strycore) [inline lutris/sysoptions.py]: Same here, don't commit commented code.

**Comment by juliotux** [comment]
Fixed commented code with 92b067983aa5430e2cc7ab297f832f8414c79d43 .

**Comment by strycore** [comment]
Thanks!

**Pull request merged**